src/docker/compose.dss-global.yaml

```diff
@@ -31,6 +31,7 @@ services:
       - HEALTHCHECK_BEFORE_START=false
 
   ws:
+    platform: linux/x86_64
```

Comment:
You're correct that it shows a warning on Mac. However, if I don't include that, then the container exits on starting with an error: https://stackoverflow.com/questions/68630526/lib64-ld-linux-x86-64-so-2-no-such-file-or-directory-error

I don't _think_ it should make compiling impossible in some cases. Can you explain how you came to that conclusion?
Not sure what the solution to this is if not that. @CRBl69 had no issue because he's on Linux already.

Reply:
Noting here as discussed offline that the issue could be solved by compiling postgres-websockets from source in `Dockerfile.ws`

Reply:
Tagging https://github.com/diogob/postgres-websockets/issues/93